package.json

```diff
@@ -18,6 +18,7 @@
     "@actions/exec": "1.1.1",
     "@actions/http-client": "2.1.0",
     "@actions/tool-cache": "2.0.1",
+    "@iarna/toml": "^2.2.5",
```

Comment:
Let's remove the `^`. We want results to be predictable in both consumption and our own CI.